modules/sdk/project-templates/project-templates-workspace/src/main/resources/archetype-resources/pom.xml

```diff
@@ -12,24 +12,12 @@
 	<version>1.0.0</version>
 	<name>${artifactId}</name>
 	<properties>
-#if (${liferayVersion.startsWith("7.0")})
-		<liferay.bom.version>7.0.6-2</liferay.bom.version>
-		<liferay.workspace.bundle.url>https://releases-cdn.liferay.com/portal/7.0.6-ga7/liferay-ce-portal-tomcat-7.0-ga7-20180507111753223.zip</liferay.workspace.bundle.url>
-#elseif (${liferayVersion.startsWith("7.1")})
-		<liferay.bom.version>7.1.3-1</liferay.bom.version>
-		<liferay.workspace.bundle.url>https://releases-cdn.liferay.com/portal/7.1.3-ga4/liferay-ce-portal-tomcat-7.1.3-ga4-20190508171117552.tar.gz</liferay.workspace.bundle.url>
-#elseif (${liferayVersion.startsWith("7.2")})
-		<liferay.bom.version>7.2.1-1</liferay.bom.version>
-		<liferay.workspace.bundle.url>https://releases-cdn.liferay.com/portal/7.2.1-ga2/liferay-ce-portal-tomcat-7.2.1-ga2-20191111141448326.tar.gz</liferay.workspace.bundle.url>
-#elseif (${liferayVersion.startsWith("7.3")})
-		<liferay.bom.version>7.3.7</liferay.bom.version>
-		<liferay.workspace.bundle.url>https://releases-cdn.liferay.com/portal/7.3.7-ga8/liferay-ce-portal-tomcat-7.3.7-ga8-20210610183559721.tar.gz</liferay.workspace.bundle.url>
-#elseif (${liferayVersion.startsWith("7.4")})
-		<liferay.bom.version>7.4.3.36</liferay.bom.version>
-		<liferay.workspace.bundle.url>https://releases-cdn.liferay.com/portal/7.4.3.36-ga36/liferay-ce-portal-tomcat-7.4.3.36-ga36-20220802152002501.tar.gz</liferay.workspace.bundle.url>
-#else
+
```

Comment:
Brian might ask about this change and want it to be set automatically like before.

If that's not possible or this is better, its fine to leave as is.

Reply:
we do not want to hard code to set liferay bom version. User will set exactly bom version when they use this template to init a liferay maven workspace project. When user use blade or pure maven archetype to create a maven workspace, he must to provide the exactly liferay bom version and we do not need to use hard code bom version any more.